scripts/spyder_win_post_install.py

```diff
@@ -21,17 +21,95 @@
 KEY_C1 = KEY_C0 + r"\command"
 
 
+# ability to run spyder-win-post-install outside of bdist_wininst installer
+# copied from pywin32-win-post-install

```

Comment:
Sorry for not being clear enough :-) Please add the link to the file, below the last line of your comment.